release.md

```diff
@@ -31,11 +25,13 @@ pagination:
 		</div>
 	</div> 
 </section>
+{%- comment -%}
 <!-- nav aria-labelledby="my-pagination">
   <h2 id="my-pagination">This is my Pagination</h2>
   <ol>
 	{%- for pageEntry in pagination.pages %}
 	    <li>item</li>
 	{%- endfor %}
   </ol>
-</nav -->
\ No newline at end of file
+</nav -->
+{%- endcomment -%}
```

Comment:
This is much better to not generate commented html. However, would not be better to just remove it this portion of code, and take it back from the git history if ever needed again?

Reply:
Sure, I will remove this before merging, thanks!